src/Analysis/Ast/Impl/Values/Collections/PythonDictionary.cs

```diff
@@ -62,7 +62,7 @@ public IReadOnlyList<IPythonCollection> Items
             _contents.TryGetValue(key, out var value) ? value : UnknownType;
 
         public override IPythonIterator GetIterator() =>
-            Call(@"iterkeys", ArgumentSet.Empty) as IPythonIterator ?? new EmptyIterator(Type.DeclaringModule.Interpreter.UnknownType);
+            Call(@"iterkeys", null) as IPythonIterator ?? new EmptyIterator(Type.DeclaringModule.Interpreter.UnknownType);
```

Comment:
null?

Reply:
ah my bad should be ArgumentSet.EmptyNoContext